image/verify/src/verifier.rs

```diff
@@ -348,6 +371,14 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
             Err(CaliptraError::IMAGE_VERIFIER_ERR_VENDOR_ECC_SIGNATURE_INVALID)?;
         }
 
```

Comment:
I had thought LMS was optional unless a fuse was set.  Is this the place to check for that fuse value and conditionally do the LMS verification?

Reply:
I believe so; future PR?